bin/plugin/commands/test-plugins.js

```diff
@@ -498,6 +502,15 @@ function doRunStandalonePluginTests( settings ) {
 		log( output );
 	} );
 
+	// Remove the ep-env overrides file.
```

Comment:
Typo: `ep-env`

Also I wonder if you should run this before running any of the tests so that this is cleaned up as soon as the environment is started?

Reply:
@joemcgill this is addressed in a recent commit. Thank you!